### PR TITLE
ismail-branch

### PR DESCRIPTION
**Pull request by ismail-zdn**
The login functionalty is added.

### CODE REVIEW

**Comment by ismail-zdn** [comment]
Hi everybody, we (Igor and I) completed the automation of the login positive scenario. 
Let's resolve the conflict then merge our branch with the master 👊

File: src/test/java/com/zerobank/pages/BasePage.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import javax.xml.xpath.XPath;
+
+public abstract class BasePage {
+
+    public BasePage() {
+        PageFactory.initElements(Driver.get(), this);
+    }
+
+    @FindBy(css="[class='icon-signin']")
+    public WebElement signInButton;
```
Comment (ismail-zdn) [inline src/test/java/com/zerobank/pages/BasePage.java:17]: This the sign in button Web Element which is in the main page.

File: src/test/java/com/zerobank/pages/LoginPage.java [hunk 2]
```diff
@@ -1,4 +1,44 @@
...
+
+    @FindBy(css = "[id='proceed-link']")
+    public WebElement proceedLinkButton;
+
+    @FindBy(css = "[class='alert alert-error']")
+    public WebElement alertError;
+
+
+    public void login() {
+        signInButton.click();
```
Comment (ismail-zdn) [inline src/test/java/com/zerobank/pages/LoginPage.java:35]: This line added to navigate to login page.

**Comment by carloslevo** [comment]
pull request accepted, mainly changes in Login Page

**Pull request merged**